Dockerfile_incl

```diff
@@ -0,0 +1,63 @@
+# syntax=docker/dockerfile:1
+
+# Dockerfile_incl is built as the `commcarehq_incl` image, and is used
+# for running a simple but complete CommCare HQ environment that
+# includes Pillowtop and Celery containers.
+
+FROM python:3.9
+MAINTAINER Dimagi <devops@dimagi.com>
+
+ENV PYTHONUNBUFFERED=1 \
+    PYTHONUSERBASE=/vendor \
+    PATH=/vendor/bin:$PATH \
+    NODE_VERSION=14.19.1
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        default-jdk \
+        wget \
+        libxml2-dev \
+        libxmlsec1-dev \
+        libxmlsec1-openssl \
+        gettext
+
+# Deletes all package sources, so don't apt-get install anything after this:
+RUN rm -rf /var/lib/apt/lists/* /src/*.deb
+
+# Install Node
+RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.gz" && \
+    tar -xzf "node-v$NODE_VERSION-linux-x64.tar.gz" -C /usr/local --strip-components=1 && \
+    rm "node-v$NODE_VERSION-linux-x64.tar.gz"
+
+RUN npm -g install \
+    yarn \
+    bower \
+    grunt-cli \
+    uglify-js
+
+# prefer https for git checkouts made by pip
+RUN git config --global url."https://".insteadOf git://
+
+WORKDIR /vendor
+RUN mkdir /sharedfiles
+RUN groupadd -r cchq && \
+    useradd -r -m -g cchq cchq && \
+    chown cchq:cchq /vendor /sharedfiles
+USER cchq:cchq
+
+#COPY requirements/prod-requirements.txt /vendor/requirements.txt
+# TODO: manage.py appears to need nose. How does prod avoid this?
```

Comment:
I don't see it. Can you elaborate how manage.py needs nose?

Reply:
I don't know how manage.py needs nose, but when I use prod-requirements.txt, ./manage.py dies, complaining that a nose library is missing. (I thought I had the traceback in my text editor but it turns out I don't. I'll run it again and see if I can get to the bottom of this.)